fuzz/client.c

```diff
@@ -65,13 +65,24 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
 
     /* This only fuzzes the initial flow from the client so far. */
     ctx = SSL_CTX_new(SSLv23_method());
+    if (ctx == NULL)
+        goto end;
 
     client = SSL_new(ctx);
+    if (client == NULL)
+        goto end;
     OPENSSL_assert(SSL_set_min_proto_version(client, 0) == 1);
     OPENSSL_assert(SSL_set_cipher_list(client, "ALL:eNULL:@SECLEVEL=0") == 1);
     SSL_set_tlsext_host_name(client, "localhost");
     in = BIO_new(BIO_s_mem());
+    if (in == NULL) {
+        goto end;
+    }
     out = BIO_new(BIO_s_mem());
+    if (out == NULL) {
+        BIO_free(in);
+	goto end;
```

Comment:
Braces are needed here, but the goto statement is not indented properly.

Reply:
It will be more rigorous next time. Thanks.